apisix/cli/ops.lua

```diff
@@ -602,6 +602,10 @@ Please modify "admin_key" in conf/config.yaml .
     end
 
     for i, r in ipairs(sys_conf["dns_resolver"]) do
+        local res, err = cli_ip.valid(r)
+        if not res then
+            util.die("invalid dns resolver address: ", r,  ", err: ", err)
```

Comment:
Would be better to ignore the unexpected resolver instead of dying of it?
And we can add a warn log for it.

Reply:
agree, fix it.